packages/xo-web/src/common/intl/locales/it.js

```diff
@@ -680,8 +680,20 @@ export default {
   // Original text: 'Resource set: {resourceSet}'
   homeResourceSet: 'Set di risorse: {resourceSet}',
 
-  // Original text: 'High Availability'
-  highAvailability: 'Alta disponibilità',
```

Comment:
Nope it's not. We don't need this parameter anymore since we let the user choose the fault tolerance. If he chooses a fault tolerance that is greater than 0, then it is automatically an high available cluster.

Reply:
I agree for `recipeHighAvailability`, but this one is `highAvailability`.

Reply:
Oh I think you're right.

I'll correct that.